### PR TITLE
Version control

### PR DESCRIPTION
**Pull request by ryanpeek**
Well, this has been a far longer and harder lesson than I hoped...I feel like it just needs a new set of eyes at this point. Hopefully it isn't terrible. Please look for ways to tighten this up, I've gone back and forth quite a bit on this lesson and just need to get it finished and move on. So much to cover and lots of concepts...which while working towards an intermediate
audience, if they've never used git before is really like teaching git from scratch.

Let me know what you think.

### CODE REVIEW

**Comment by richpauloo** [comment]
Overall, this is a great lesson with a lot of content. It would be awesome if we could get through this entire lesson because it truly demonstrates a real life workflow! Great job Ryan.

I made editorial changes throughout to improve clarity, and minor formatting. I removed larger width layout options in chunks later in the document because they prevented rather long asides from printing, and led to large blank white spaces that broke up the document which made it hard to follow the narrative.

One thing I found somewhat confusing was that "Main" is a person, and also a branch. The only purpose Main serves as a person is that they create the branch and add collaborators -- beyond that Main doesn't contribute changes. I would consider deleting Main as a person from the figs (and the one reference to Main as a person), and just saying that "Jo" creates the "main" branch and introduces Mo as a collaborator. Just a thought. I think it might reduce potential confusion between Main the person and the branch, and focus attention on the main contributors in the story (Mo and Jo).

**Comment by ryanpeek** [comment]
Sounds good and makes sense, I had plans to pull in `Main` as the supervisor that reviewed changes and merged the pull requests, but I agree, it makes it really long and a bit confusing as there isn't a part for Main currently. I'll modify figures and just remove `Main` the person, but keep the `main` branch as suggested.

Quick side request, for PR's and review, if you can just push commits for the `.Rmd` and not the `html` and `search.json` (unless there's a specific need or change), it should keep things cleaner and avoid merge conflicts as we move forward. I can always knit in branch to preview, and then we can keep these "temp" files isolated to just the final web build in develop or main. Happy to chat more if you have thoughts on this.

**Comment by richpauloo** [comment]
Just checked out the new lesson and I think it's much clearer with that cloud Github. Since people have experience with Drive, Box, Dropbox, and so on, I think this will make things really clear for people. Thanks for making that change!

Also, my bad on committing search.json and knitted m_{name}.html files. I can see how that may cause conflicts, and won't do that again until we're ready for the final merge into main. Thanks for bringing it up. =)

**Comment by ryanpeek** [comment]
no worries! And thanks for the helpful revision suggestions...it's much cleaner now. Onward! 😉
